packages/integration-karma/test/component/LightningElement.toString/index.spec.js

```diff
@@ -11,6 +11,6 @@ it('should rely on the constructor name', () => {
 if (process.env.COMPAT !== true) {
     it('should fallback to BaseLightningElementConstructor if constructor has no name', () => {
         const elm = createElement('x-anonymous', { is: Anonymous });
-        expect(elm.getToString()).toBe('[object BaseLightningElementConstructor]');
+        expect(elm.getToString()).toBe('[object LightningElementConstructor]');
```

Comment:
This is technically not a breaking change but it's an observable one...we should keep it in mind in case it causes issues.

Reply:
If we are renaming the base class constructor name, why not renaming it to `LightningElement`? I was always strange to me that `LightningElement.name !== 'LightningElement'`.

What do you think @abdulsattar @ekashida ?

Reply:
It requires a couple of extra `@ts-ignore`s due to  the `CustomElementConstructor` not being defined and `prototype` being readonly. Please see the new commit.